### ISSUE TITLE
[BUG] No processing of Lombok annotations with values in Spring Tool Suite 4.22.0

### ISSUE THREAD

**Issue by explorer566** [projectlombok/lombok#3630]
**Describe the bug**
Lombok annotations with values don't get processed. This happens with Lombok 1.18.30 on Windows 10 with Java 17 in Eclipse Spring Tool Suite 4.22.0 and was working in previous versions of STS.

**Working example**

 ```
import java.io.Serializable;
 import java.time.LocalDate;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;

@Getter
@Setter
@EqualsAndHashCode
@ToString
public class Test implements Serializable {

	private String myString;

	private LocalDate myDate;
}

```
Generates getter, setter, equals, hashCode und toString methods.

**Non working example**

 ```
import java.io.Serializable;
 import java.time.LocalDate;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;

@Getter
@Setter
@EqualsAndHashCode(of = { "myString", "myDate" })
@ToString(doNotUseGetters = true)
public class Test implements Serializable {

	private String myString;

	private LocalDate myDate;
}
```
Generates only getter and setter methods.

**Version info**
 - Lombok 1.18.30
 -  Windows 10, Java 17, Eclipse Spring Tool Suite 4.22.0

**Comment by martinlippert** [comment]
Most likely related to https://github.com/projectlombok/lombok/issues/3620, since the Spring Tools 4.22.0 for Eclipse are based on Eclipse 2024-03.

**Comment by explorer566** [comment]
Problem is fixed with Edge version 1.18.31 from 2024-1-12
